stl/inc/concepts

```diff
@@ -9,7 +9,9 @@
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
 
-#ifdef __cpp_lib_concepts
+#ifndef __cpp_lib_concepts
+#pragma message("The contents of <concepts> are only available with C++20 or later.")

```

Comment:
This (and `<ranges>`) was intentionally "C++20 concepts support" to avoid clang users becoming confused: "I'm compiling with `/std:c++latest` but the compiler still doesn't see the contents of the header!". (clang and EDG (the IntelliSense frontend) have C++20 modes, but don't yet implement C++20 concepts.) This is also why these headers (and much of `<xutility>`) are guarded with `__cpp_lib_concepts` instead of simply `_HAS_CXX20`.

I these should stay "C++20 concepts support" for now.

Reply:
Makes sense.